### DISCUSSION TITLE
adding funding information

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#32] [Q&A]
How do I add funding information to a dandiset?

![image](https://user-images.githubusercontent.com/844306/135909331-70b174ed-3166-4d1f-9b49-0190e8110b7b.png)

**Comment by satra** [comment]
Funding information is added in the contributor metadata section. contributors can be people or organizations.

**Comment by bendichter** [comment]
Thanks @satra. What should we do if there are multiple organizations associated with a grant number or an organization associated with multiple grant numbers? Both are true for dandiset 8.

**Reply by satra** [reply]
for grant numbers, typically the primary organization that receives the award is listed as would be the case on reporter.nih.gov. 
 but in dandiland you can repeat grant+institution as many times as you want.

**Reply by bendichter** [reply]
got it, thanks
